### ISSUE TITLE
[suggestions] ship controle hotkeys

### ISSUE THREAD

**Issue by KINGrimson** [tsunamayo/Starship-EVO#3378]
### Short Description of the issue:
my idea is that when in a ship and when you press something like l for light cruise you can have a hotkey trigger on that input seperetly from light cruise you can have something move or a key of movements to indicate you are switching flight modes
(light cruise would still work without the hot key) 
kinda like this 

press l > light cruise engages 
           > hotkey on input l > does something 
...

### Steps to reproduce the issue:

...

### Environment
- Starship EVO version: 
- Graphics Card: 

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`

**Comment by tsunamayo** [comment]
Yes this is planned, it will be called the event gate and will replace (or extend) the hotkey gate.
